android/sdl_android/src/androidTest/java/com/smartdevicelink/test/rpc/datatypes/GateStatusTest.java

```diff
@@ -0,0 +1,66 @@
+package com.smartdevicelink.test.rpc.datatypes;
+
+import com.smartdevicelink.marshal.JsonRPCMarshaller;
+import com.smartdevicelink.proxy.rpc.GateStatus;
+import com.smartdevicelink.proxy.rpc.Grid;
+import com.smartdevicelink.proxy.rpc.RoofStatus;
+import com.smartdevicelink.proxy.rpc.enums.DoorStatusType;
+import com.smartdevicelink.test.JsonUtils;
+import com.smartdevicelink.test.TestValues;
+import com.smartdevicelink.test.Validator;
+
+import junit.framework.TestCase;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.Hashtable;
+import java.util.Iterator;
+
+public class GateStatusTest extends TestCase {
+    private GateStatus msg;
+
+    @Override
+    public void setUp() {
+        msg = new GateStatus(TestValues.GENERAL_GRID, TestValues.GENERAL_DOOR_STATUS_TYPE);
+        msg.setStatus(TestValues.GENERAL_DOOR_STATUS_TYPE);
```

Comment:
@kostyaBoss You can remove line 26  (`msg.setStatus(...)`) since you are already setting the status using the `DoorStatus` constructor.